Update the postgreql version constrains to match the version available in postgresql chart.
---

diff --git a/bitnami/keycloak/Chart.yaml b/bitnami/keycloak/Chart.yaml
index b5e9dc9..ba34e72 100644
--- a/bitnami/keycloak/Chart.yaml
+++ b/bitnami/keycloak/Chart.yaml
@@ -15,7 +15,7 @@ dependencies:
 - condition: postgresql.enabled
   name: postgresql
   repository: oci://registry-1.docker.io/bitnamicharts
-  version: 16.x.x
+  version: 17.x.x
 - name: common
   repository: oci://registry-1.docker.io/bitnamicharts
   tags:
